frontend/test/playwright/e2e/load-more.spec.ts

```diff
@@ -12,7 +12,7 @@ import { AUDIO, IMAGE, SupportedMediaType } from "~/constants/media"
 import {
   collectAnalyticsEvents,
   expectEventPayloadToMatch,
-} from "../utils/analytics"
+} from "~/utils/analytics"
```

Comment:
```suggestion
} from "~~/test/playwright/utils/analytics"
```
Sorry, I didn't see that this was in tests